modules/mtl/arrow-mtl-data/src/main/kotlin/arrow/mtl/Kleisli.kt

```diff
@@ -15,8 +15,11 @@ import arrow.typeclasses.Monad
  */
 typealias KleisliFun<F, D, A> = (D) -> Kind<F, A>
 
+@Deprecated("Kleisli#run can be easily mistaken with the standard library extension function", ReplaceWith("runK(d)"))
 fun <F, D, A> KleisliOf<F, D, A>.run(d: D): Kind<F, A> = fix().run(d)
 
+fun <F, D, A> KleisliOf<F, D, A>.runK(d: D): Kind<F, A> = fix().run(d)
```

Comment:
can we make this `fun operator invoke`?

Reply:
I'd rather we didn't, at least for this first version. Automatic replacement can cause syntactic errors.